packages/account-v2/src/pages/TwoFactorAuthentication/TwoFactorAuthenticationDisabled.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { ComponentProps, useEffect } from 'react';
+import { twMerge } from 'tailwind-merge';
+import { useAuthorize, useTwoFactorAuthentication } from '@deriv/api-v2';
+import { Loader, Text } from '@deriv-com/ui';
+import { Timeline } from '../../components/Timeline';
+import { TWO_FACTOR_AUTHENTICATION_URLS } from '../../constants';
+import { TwoFactorAuthenticationForm } from '../../containers/TwoFactorAuthenticationForm';
+import { TwoFactorAuthenticationArticle } from './TwoFactorAuthenticationArticle';
+import { TwoFactorAuthenticationQRCode } from './TwoFactorAuthenticationQRCode';
+
+const TwoFactorAuthenticationLink = ({ children, className = '', ...rest }: ComponentProps<'a'>) => {
+    return (
+        <a
+            {...rest}
+            className={twMerge('hover:underline font-bold', className)}
+            rel='noopener noreferrer'
+            target='_blank'
+        >
+            {children}
+        </a>
+    );
+};
+
+export const TwoFactorAuthenticationDisabled = () => {
+    const { data: authorizeData } = useAuthorize();
+    const { email = '' } = authorizeData ?? {};
+    const { data: secretKeyData, isLoading, mutate } = useTwoFactorAuthentication();
+    const { secretKey } = secretKeyData ?? {};
+    useEffect(() => {
+        mutate({ totp_action: 'generate' });
+    }, [mutate]);
+
+    return (
+        <div className='flex flex-1 w-full h-full sm:flex-col-reverse sm:col-span-2'>
+            <section className='py-0 px-[14px]'>
+                <Text
+                    align='start'
+                    as='h2'
+                    className='ml-[12px] mb-[15px] text-[16px] text-system-light-prominent-text leading-normal sm:mr-1.2rem'
+                    weight='bold'
+                >
+                    How to set up 2FA for your Deriv account
+                </Text>
+                <Timeline className='pt-0 px-[14px] pb-[16px] m-[12px] w-full' lineHeight='xs'>
+                    <Timeline.Item
+                        itemTitle={
+                            <Text align='start' className='leading-tight text-[14px] text-system-light-prominent-text'>
```

Comment:
Can u use size property for `Text` component

Reply:
Yeah @likhith-deriv, but as I checked, the text size in `@deriv-com/ui` comes with line height that doesn't match what's in `account-v1`. That's why I used the tailwind version.